packages/appstore/src/components/wallet-header/wallet-header.scss

```diff
@@ -55,66 +55,6 @@
 
         &-buttons {
             display: flex;
```

Comment:
I think we can remove this to be honest.

Reply:
what happens if you add `flex` in the `wallet-header` instead?

Reply:
@aizad-deriv I'm using flex for wallet title and buttons container named `wallet-header__description `, there are 2 divs inside: `title` and `buttons`, and I need `flex` for `buttons` container.

Reply:
@sergei-deriv okay then it should be fine 😃 👍🏼